system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -6217,7 +6217,7 @@ class CLdapClient : implements ILdapClient, public CInterface
         StringBuffer resName(queryDfsXmlBranchName(DXB_Internal));
         resName.append("::").append(username);
         Owned<ISecResource> resource = new CLdapSecResource(resName.str());
-        if (!addResource(RT_FILE_SCOPE, user, resource, PT_ADMINISTRATORS_AND_USER, m_ldapconfig->getResourceBasedn(RT_FILE_SCOPE)))
+        if (!addResource(RT_FILE_SCOPE, user, resource, PT_ADMINISTRATORS_ONLY, m_ldapconfig->getResourceBasedn(RT_FILE_SCOPE)))
```

Comment:
Don't think this place applies, because this is within an operation specifically for a certain user.